pkg/version/version.go

```diff
@@ -33,8 +33,9 @@ var (
 	// since an app version must uniquely determine the state of the schema.
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
-	Version    = "0.1.2-dev.0"
-	CommitHash = ""
+	Version                     = "0.1.2-dev.0"
+	CommitHash                  = ""
+	VersionToUpgradeFromInTests = "0.1.0"
```

Comment:
oy that name. What do you think of `EarliestCompatibleVersion` or `EarliestTestedVesrsion` or, dare I say it, `EarliestSupportedVersion`, or something like that?

Reply:
`EarliestUpgradeableVersion`?